cli/src/main/kotlin/commands/RequirementsCommand.kt

```diff
@@ -100,28 +103,66 @@ class RequirementsCommand : CliktCommand(name = "requirements", help = "List the
             }
         }
 
+        var statusCode = 0
+
         allTools.forEach { (category, tools) ->
             println("${category}s:")
+
             tools.forEach { tool ->
-                // TODO: State which version was found, and whether it could be bootstrapped, but that requires
-                //       refactoring of CommandLineTool.
+                // TODO: State whether a tool can be bootstrapped, but that requires refactoring of CommandLineTool.
                 val message = buildString {
-                    if (tool.isInPath()) append("\t* ") else append("\t- ")
+                    val (prefix, suffix) = if (tool.isInPath()) {
+                        try {
+                            val actualVersion = tool.getVersion()
+                            try {
+                                if (tool.getVersionRequirement().isSatisfiedBy(actualVersion)) {
+                                    Pair("\t* ", "Found version $actualVersion.")
+                                } else {
+                                    statusCode = statusCode.or(2)
```

Comment:
Why not write it as `statusCode or 2` as `or` is an `infix fun`? For me this would make it easier to not confuse it with the semantics of e.g. `orEmpty()`.

Reply:
For some reason the unfix version was not found when I tried, but I'll check again.

Reply:
Done.